libs/libvtrutil/src/generate.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+for filename in *.h
```

Comment:
Seems like this is a one off use script? If so it probably shouldn't be checked in.

Otherwise, we should give more documentation to clarify what this is for.

Reply:
Thanks for catching that. This file shouldn't be upstreamed as it was only used to generate an initial version for the .rst file.